features/google/src/model/importers/access-log-importer.js

```diff
@@ -40,7 +40,7 @@ export default class AccessLogImporter {
     async import({ zipFile, facebookAccount: googleAccount }) {
         const entries = await relevantZipEntries(zipFile);
         const accessLogEntries = entries.filter(({ path }) => {
-            return matchRegex(path, this);
+            return matchRegex(path, this.constructor.name);
```

Comment:
Is this a different fix?

Reply:
Yea, kinda, but related. `matchRegex` will only need the name, which is used to index the regex array. Otherwise, it would have needed to import all classes that are going to be matched, which makes no sense.

Reply:
I would not call it a fix. It is a slight modification to allow for easy testing.

Reply:
> I would not call it a fix. It is a slight modification to allow for easy testing.

Yep, it was a fix. You don't need to pass classes (which you need to import) when all you need is the class name for indexing the array.